backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/incremental/DiagramLayoutTests.java

```diff
@@ -95,6 +102,11 @@ public Optional<IRepresentationDescription> findById(IEditingContext editingCont
         return new TestDiagramCreationService(this.objectService, representationDescriptionSearchService, layoutService);
     }
 
+    @BeforeEach
+    public void setup() {
+        this.targetObjectIdToNodeId.clear();
+    }
```

Comment:
We worked quite hard to make sure that our code is as stateless as possible so we won't make our tests stateful. Do not rely on a stateful property in the test and remove this targetObjectIdToNodeId property.

Reply:
Removed